components/service/fretboard/src/test/java/mozilla/components/service/fretboard/FretboardTest.kt

```diff
@@ -431,4 +433,55 @@ class FretboardTest {
         val fretboard = Fretboard(source, storage)
         fretboard.updateExperiments()
     }
+
+    @Test
+    fun getUserBucketTest() {
```

Comment:
Sorry, just a small one!
Can we make this test name standard as the other ones.
`testUserBucket(`)